server.js

```diff
@@ -41,6 +41,33 @@ app.get('/api/v1/races/:id', async (request, response) => {
   }
 });
 
+app.get('/api/v1/races/:id/classes', async (request, response) => {
+  try {
+    const raceClasses = await database('raceClasses').where('race_id', request.params.id).select();
+
+    if (!raceClasses.length) {
+      return response.status(404).json({
+        error: `Could not find any class for this race`
+      });
+    }
+
+    const classes = await getClasses(raceClasses);
+
+    return response.status(200).json(classes);
```

Comment:
4. Get specific classes from `classes` table in DB and if it's successful classes array is returned in response with status code `200`